Database/Updates/World/01-06-19-2017-alterPostionTable.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE `ace_world`.`ace_position`   
+  DROP COLUMN `landblock` ,
+  ADD COLUMN `landblock` INT(5) UNSIGNED AS ( ((landblockraw >> 16) & 0xFFFF)) VIRTUAL,
+  DROP COLUMN `cell` ,
+  ADD COLUMN `cell` INT(5) UNSIGNED AS ( (CONV(RIGHT(LPAD(HEX(`landblockRaw`),8,'0'),4),16,10)) ) VIRTUAL, 
```

Comment:
but this is so painful!  do we even have a use case for needing cell as a separate column?  if not, just drop it.